mars/dataframe/core.py

```diff
@@ -1268,6 +1269,96 @@ def to_frame(self, name=None):
         name = name or self.name or 0
         return dataframe_from_tensor(self, columns=[name])
 
+    def between(self, left, right, inclusive="both"):
+        """
+        Return boolean Series equivalent to left <= series <= right.
+        This function returns a boolean vector containing `True` wherever the
+        corresponding Series element is between the boundary values `left` and
+        `right`. NA values are treated as `False`.
+        Parameters
```

Comment:
Please pay attention to the blank lines in docs. I will skip to mention the other part in the docs